### PR TITLE
fix(bot-handler-avs): Add compatibility with Node.js 14

### PR DESCRIPTION
**Pull request by ffflorian**
## Pull Request Checklist

- [x] My code is covered by tests
- [x] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/bot-handler-avs/package.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "dependencies": {
     "@types/webrtc": "0.0.26",
-    "@wireapp/avs": "6.0.46",
-    "wrtc": "0.4.5"
+    "@wireapp/avs": "6.0.46"
```
Comment (bennyn) [inline packages/bot-handler-avs/package.json:4]: Let's update to the latest AVS 6.2 here.

Comment (ffflorian) [reply packages/bot-handler-avs/package.json]: I'd rather not change other dependencies here.

File: packages/bot-handler-avs/src/AVSHandler.ts [hunk 2]
```diff
@@ -33,7 +33,13 @@ import {
 import axios from 'axios';
 import type {CallMessage} from '@wireapp/core/dist/conversation/message/OtrMessage';
 
-const wrtc = require('wrtc');
+let wrtc;
+
+try {
+  wrtc = require('wrtc');
+} catch (error) {
+  throw new Error('wrtc was not installed. Please use Node.js <= 13.0');
```
Comment (bennyn) [inline packages/bot-handler-avs/src/AVSHandler.ts]: Not required anymore with wrtc 0.4.6

Comment (ffflorian) [reply packages/bot-handler-avs/src/AVSHandler.ts]: Done :)

**Pull request merged**